apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -229,6 +231,16 @@ class ShareAllowedDialog extends React.Component {
     this.props.onUnpublish(this.props.channelId);
   };
 
+  createReplayVideo = () => {
+    fetch("https://dance-api.code.org/render", {
```

Comment:
Great use of `fetch`!  Should we handle the promise rejection, in case the service is down or times out?

Reply:
I'm probably going to implement some kind of retry in the final version that depends on S3